workspace/apps/pidp/src/app/features/portal/portal.page.ts

```diff
@@ -80,6 +93,38 @@ export class PortalPage implements OnInit, AfterViewInit {
     this.alerts$ = this.resource
       .getProfileStatus(this.partyService.partyId)
       .pipe(map((profileStatus) => profileStatus?.alerts ?? []));
+    if (this.previousUrl.split('/').includes(Constants.newUserURL)) {
+      this.showNewUserPopup();
+    }
+  }
+
+  private showNewUserPopup(): void {
+    const data: DialogOptions = {
+      title: 'Account information completed',
+      bottomBorder: false,
+      titlePosition: 'center',
+      bodyTextPosition: 'center',
+      component: HtmlComponent,
+      data: {
+        content:
+          'Thank you for providing OneHealthID with the information needed at this time, now you can explore OneHealthID to its full potential, based on your provided information.',
+      },
+      imageSrc: '/assets/images/online-marketing-hIgeoQjS_iE-unsplash.jpg',
+      imageType: 'banner',
+      width: '30rem',
+      height: '26rem',
+      cancelHide: true,
+      actionHide: true,
+      imageSizeFull: true,
+      titleMarginTop: true,
+      closeButton: true,
+      class: 'new-dialog-container',
+    };
```

Comment:
As the dialogoptions are defined in different components for different dialogs can we define dialogoptionsconstructor helper function and based on the purpose of the dialog it will construct the dialogoptions. So that components will be separated from the dialog construction logic. If some of the values are same across the dialogs then we can define only once rather than re-defining.